src/integration/TestCase.php

```diff
@@ -117,7 +124,25 @@ protected function extension(string ...$extensions)
     }
 
     /**
-     * @var string[]
+     * @var array
+     */
+    protected $config = [];
+
+    /**
+     * Some Flarum code depends on config.php values. Flarum doesn't
+     * offer a way to set them at runtime, so this method lets you
+     * add/override them before boot.
+     * 
+     * Note that this method will have no effect if called after the
+     * application is booted.
+     */
+    protected function config(string $key, $value)
```

Comment:
How would we go about setting something like `'headers' => ['referrerPolicy' => 'strict-origin']` with this function. Looking at it right now and the test you've written this example isn't possible.

Reply:
Use a dot separated syntax. Iirc, shouldn't Arr::set do a deep assignment?

Reply:
Ah yes you're correct, my quick glance at the code failed to see the other method that actually set's it using `Arr::set()`